src/mui/input/SelectManyInput.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component, PropTypes } from 'react';
+import ChipInput from 'material-ui-chip-input';
+import FieldTitle from '../../util/FieldTitle';
+
+/**
+ * An Input component for a array
+ *
+ * @example
+ * <SelectManyInput source="first_name" />
+ *
+ * Pass possible options as an array of objects in the 'choices' attribute.
+ * @example
+ * const choices = [ 'Male', 'Female' ];
```

Comment:
I don't agree with the addition of this `type` prop. It complicates things IMO. Keep to the same API as `SelectInput` and remove the `type` prop.